mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/KeyRegistrationHandler.java

```diff
@@ -156,20 +155,20 @@ Registration register(MailboxListener listener, RegistrationKey key) {
 
         return localListenerRegistry.getLocalMailboxListeners(registrationKey)
             .filter(listener -> !isLocalSynchronousListeners(eventBusId, listener))
-            .flatMap(listener -> Mono.fromRunnable(Throwing.runnable(() -> executeListener(listener, event, registrationKey)))
-                .doOnError(e -> structuredLogger(event, registrationKey)
-                    .log(logger -> logger.error("Exception happens when handling event", e)))
-                .onErrorResume(e -> Mono.empty())
-                .then())
+            .flatMap(listener -> executeListener(listener, event, registrationKey))
             .subscribeOn(Schedulers.elastic())
```

Comment:
This could be redundant with the ReactiveMailboxListenerWrapper. Could it cause some slowness?